hs_core/views/__init__.py

```diff
@@ -653,6 +767,221 @@ def create_resource(request, *args, **kwargs):
 
 
 @login_required
+def create_user_group(request, *args, **kwargs):
+    group_form = GroupForm(request.POST, request.FILES)
+    if group_form.is_valid():
+        try:
+            new_group = group_form.save(request)
+            messages.success(request, "Group creation was successful.")
+            return HttpResponseRedirect(reverse('group', args=[new_group.id]))
+        except IntegrityError as ex:
+            if group_form.cleaned_data['name'] in ex.message:
+                message = "Group name '{}' already exists".format(group_form.cleaned_data['name'])
+                messages.error(request, "Group creation errors: {}.".format(message))
+            else:
+                messages.error(request, "Group creation errors:{}.".format(ex.message))
```

Comment:
@pkdash I get why @mauriel133 wanted it this way since the user may just input a duplicate group name, in which case we should just inform users that this group already exists. Is it better that we check whether group exists before saving it, and if yes, return group already exist message instead, and if not, we save it, and if there is integrity error while saving, we print out exception messages since integrity error is more broad than just duplicate record?